packages/macros/tests/babel/import-sync.test.ts

```diff
@@ -31,7 +31,7 @@ describe('importSync', function () {
       import { importSync as i } from '@embroider/macros';
       i('foo');
       `);
-      expect(code).toMatch(/require\(['"]foo['"]\)/);
```

Comment:
why this extra `\)`?

Reply:
its a mistake, it shouldn't here. 

we want `require("foo")` not `require("foo"))`